org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/console/SonarLintConsole.java

```diff
@@ -69,18 +64,31 @@ protected void init() {
     getDebugStream().setColor(debugColor);
   }
 
-  public void closeConsole() {
-    IConsoleManager manager = ConsolePlugin.getDefault().getConsoleManager();
-    manager.removeConsoles(new IConsole[] {this});
+  @Override
+  protected void init() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1186)